articles/privacy-art-collection.md

```diff
@@ -0,0 +1,35 @@
+|   |   |
+|---|---|
+| Title               | Privacy Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+↑ Add more table rows for more social media links. See [this completed article](articles/simple-payment-verification.md) for a real example.
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+Pre-discussion, research and questions
+
+---
+
+# Article Title
+
+## Art sample + Description
+
+## Art Piece 1 + Description
+
+## Art Piece 2 + Description
+
+## Art Piece 3 + Description
+
+## Art Piece 4 + Description
```

Comment:
Decred's coinjoin style privacy system (coinjoin++) is integrated into its governance ticket system.
Participants of the governance system constantly buy tickets time-locking their coins in the process, all those constant transactions gives the decred coinjoin system a much higher anonymity set then most other coinjoin style systems in the market
![decred privacy system 3d](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/f9b57af0-2695-407e-97ff-6bb1f3a73135)